src/Engine/Manager/RenderManager.cpp

```diff
@@ -327,15 +341,15 @@ Component::Animation* RenderManager::CreateAnimation() {
 
 Component::Animation* RenderManager::CreateAnimation(const Json::Value& node) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetAnimations' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)